pkg/api/openshiftcluster.go

```diff
@@ -88,6 +88,8 @@ type Properties struct {
 	SSHKey            SecureBytes  `json:"sshKey,omitempty"`
 	AdminKubeconfig   SecureBytes  `json:"adminKubeconfig,omitempty"`
 	KubeadminPassword SecureString `json:"kubeadminPassword,omitempty"`
+	ACRTokenName      SecureString `json:"acrTokenName,omitempty"`
```

Comment:
can use a string for this.

I wonder about something like `RegistryProfiles []RegistryProfile` where a RegistryProfile struct includes an enum type (ACR/RH), identifier, username and password.